edx_exams/apps/core/email.py

```diff
@@ -0,0 +1,61 @@
+"""
+Handles sending emails to users about proctoring status changes.
```

Comment:
#NIT We are going to put all email sending functions into this file, correct? Then I would not be so specific on this comment. This comment is more for the function below, right?